quinn/functions.py

```diff
@@ -95,3 +95,248 @@ def array_choice(col):
 @F.udf(returnType=ArrayType(StringType()))
 def regexp_extract_all(s, regexp):
     return None if s == None else re.findall(regexp, s)
+
+
+from pyspark.sql.types import StructType, ArrayType, MapType
+from pyspark.sql import DataFrame
+from pyspark.sql import functions as F
+import re
+from typing import Dict, List
+from pathlib import Path
+
+def sanitize_column_name(name: str, replace_char: str = "_") -> str:
+    """
+    Sanitizes column names by replacing special characters with the specified character.
+    
+    Args:
+        name (str): The original column name.
+        replace_char (str, optional): The character to replace special characters with. Defaults to '_'.
+    
+    Returns:
+        str: The sanitized column name.
+    """
```

Comment:
Are we able to change docstrings format to `sphinx`?

Reply:
I dont know sphinx.. but is it ok now?

Reply:
Looks great! Thank you!